pages/index.tsx

```diff
@@ -37,17 +42,23 @@ ReactDOM.render(html\`<\${Editor} />\`, document.getElementById('root'));
 
 let Editor = () => {
   const { containerRef } = useMonacoEditor({
-    plugins: ['prettier', 'typings'],
+    languages: ['javascript', 'typescript'],
+    plugins: [plugins.prettier({ semi: false }), 'typings'],
```

Comment:
Which API would you prefer:
```
plugins: [plugins.prettier({ semi: false }), 'typings']
```

or 
```
plugins: [['prettier', { semi: false }], 'typings']
```

(something like babel's plugins)

Getting typescript hinting is probably important too

Reply:
I like the second one. Same syntax as babel [plugin options](https://babeljs.io/docs/en/plugins/#plugin-options).

```js
  "plugins": [
    "pluginA",
    [ "pluginB", { ...options } ]
  ]
```